### DISCUSSION TITLE
Exosuit walk speed penalty rebalance

### DISCUSSION THREAD

**Discussion by MrBandoIruka** [PeasantFoundry/Mechtrauma#80] [Ideas]
If the exosuit is designed to carry heavy loads, how about removing its movespeed penalty whilst carrying items such as barrels, drive gears, turbine blades, etc. It will still be slow to move around in the exosuit like always, but faster than carrying heavy loads unassisted. By that same vein, a slight movespeed penalty when using a hand truck.

**Comment by PeasantFoundry** [comment]
I'm mad at Exosuits because they can climb ladders. Same problem with the hand truck. I deeply wish I could stop handtrucks and exosuits from using ladders so that people have to use item elevators to move heavy items between floors. 

Having said that, your suggestion isn't a bad one.

**Reply by MapleWheels** [reply]
The item elevator mechanic is kind of garbage honestly and needs are proper solution. Actual elevators are sub-design-specific so it's not fair to expect the player to have one readily available.

Exosuits probably just need a slightly higher speed penalty overall and remove the carry speed limitation. If you nerf it too much, people just won't use it and it'll feel bad for any player who's class has the exosuit perk.
